### ISSUE TITLE
Telescope flush output of last command.

### ISSUE THREAD

**Issue by mathieutu** [laravel/telescope#473]
Hi, 
in `\Laravel\Telescope\Watchers\CommandWatcher::recordCommand` method, there is a `app(Kernel::class)->output()` call: 
https://github.com/laravel/telescope/blob/502d400945b152155cc58792edef758717c2edec/src/Watchers/CommandWatcher.php#L40

This `output` method makes a call behind the scene to the `\Symfony\Component\Console\Output\BufferedOutput::fetch`, which clear the content of the buffer.

So if you try to access the content of this buffer later, it will be empty.

To sum up, because you can't call `output` more that once: 
```php
Artisan::call('help');
return Artisan::output();
```
will return an empty string if you have telescope.

One solution could be to extend the BufferedOutput, and add a new method that will not clear the content. If you are ok with that I could do it in a PR.

**Comment by paras-malhotra** [comment]
Sounds legit, I think this needs a PR.

**Comment by themsaid** [comment]
I've removed this call from the watcher since we're not displaying it currently.

**Comment by mathieutu** [comment]
Thanks. 
Here is the commit, for history: 
https://github.com/laravel/telescope/commit/48203746bf6fac76454058f3612a740db9b4b4f3
